utils/firestore.js

```diff
@@ -184,6 +185,210 @@ const updateParticipantData = async (id, data) => {
+            fieldMapping.dataDestruction.consentFirstName,
+            fieldMapping.dataDestruction.consentMiddleName,
+            fieldMapping.dataDestruction.consentLastName,
+            fieldMapping.dataDestruction.consentSuffixName,
+            fieldMapping.dataDestruction.userProfileNameFirstName,
+            fieldMapping.dataDestruction.userProfileNameMiddleName,
+            fieldMapping.dataDestruction.userProfileNameLastName,
+            fieldMapping.dataDestruction.userProfileNameSuffixName,
+            'query',
+            fieldMapping.autogeneratedConsentDate,
+            fieldMapping.participantMap.consentFormSubmitted,
+            fieldMapping.dataDestruction.informedConsentDateSigned,
+            fieldMapping.dataDestruction.informedConsentVersion,
+            fieldMapping.dataDestruction.hipaaAuthorizationDateSigned,
+            fieldMapping.dataDestruction.hipaaAuthorizationFlag,
+            fieldMapping.dataDestruction.hipaaAuthorizationVersion,
+            fieldMapping.dataDestruction.firebaseAuthenticationEmail,
+            fieldMapping.firebaseAuthenticationFirstAndLastName,
+            fieldMapping.authenticationPhone,
+            fieldMapping.signInMechanism,
+            fieldMapping.preferredLanguage,
+            fieldMapping.preferredName,
+            fieldMapping.dataDestruction.birthMonth,
+            fieldMapping.dataDestruction.birthDay,
+            fieldMapping.dataDestruction.birthYear,
+            fieldMapping.dataDestruction.dateOfBirth,
+            fieldMapping.cellPhone,
+            fieldMapping.homePhone,
+            fieldMapping.otherPhone,
+            fieldMapping.prefEmail,
+            fieldMapping.additionalEmail1,
+            fieldMapping.additionalEmail2,
+            fieldMapping.additionalEmail3,
+            fieldMapping.address1,
+            fieldMapping.address2,
+            fieldMapping.city,
+            fieldMapping.state,
+            fieldMapping.zip,
+            fieldMapping.canWeVoicemailMobile,
+            fieldMapping.canWeVoicemailHome,
+            fieldMapping.canWeVoicemailOther,
+            fieldMapping.canWeText,
+            fieldMapping.prefContactMethod,
+            fieldMapping.haveYouEverBeenDiagnosedWithCancer,
+            fieldMapping.whatYearWereYouDiagnosed,
+            fieldMapping.whatTypeOfCancer,
+            fieldMapping.anyCommentsAboutYourCancerDiagnosis,
+            fieldMapping.derivedAge,
+            fieldMapping.dataDestruction.userProfileSubmittedFlag,
+            fieldMapping.autogeneratedProfileSubmittedTime,
+            fieldMapping.participantMap.consentFormSubmitted,
+            fieldMapping.verificationStatus,
+            fieldMapping.autogeneratedSignedInTime,
+            fieldMapping.autogeneratedVerificationStatusUpdatedTime,
+            // These are not in either the field mapping or the concept ID DB lookup, but are generated on the user during the creation process
+            983784715, 
```

Comment:
We should reach out to the Analytics team to determine what these fields represent

Reply:
Per conversation, these are deprecated fields. I will be leaving them in for now in order to make sure the data is identical, with a ticket to be filed down the line to remove them from the codebase.